pasta_eln/miscTools.py

```diff
@@ -59,7 +59,7 @@ def outputString(fmt:str='print', level:str='info', message:str='') -> str:
   if 'text' in fmt:
     return txtOutput
   if fmt=='html':
-    colors = {'info':'black','error':'red','warning':'orange','ok':'green','okish':'blue','unsure':'magenta'}
+    colors = {'info':'black','error':'red','warning':'orangered','ok':'green','okish':'blue','unsure':'darkmagenta'}
```

Comment:
orangered to orange-red?

Reply:
It is a named color in matplot lib, I did not choose it
 https://matplotlib.org/stable/gallery/color/named_colors.html